spec/master_spec.js

```diff
@@ -0,0 +1,160 @@
+var noop = require("lodash.noop"),
+    cluster = require("cluster"),
+    runMasterJasmine = require("../lib/master");
+
+describe("master", function() {
+  beforeEach(function() {
+    var fakeRunJasmine = this.fakeRunJasmine = jasmine.createSpy("fakeRunJasmine");
+    spyOn(require("module"), "_load").and.callFake(function() {
+      if (arguments[0] === "./run") {
+        return fakeRunJasmine;
+      } else {
+        return this._load.and.originalFn.apply(this, arguments);
+      }
+    });
+
+    var fakeSend = this.fakeSend = jasmine.createSpy("fakeSend");
+    spyOn(cluster, "fork").and.returnValue({
+      send: fakeSend
+    })
+
+    // this.fakeJasmine = jasmine.createSpyObj('jasmine', ["execute", "configureDefaultReporter", "addSpecFiles"])
+    // this.fakeJasmine.specFiles = [
+    //   "./master_spec.js",
+    //   "./worker_spec.js"
+    // ];
+  });
+
+  it("massege callback 'jasmineDone' should send and kill properly", function() {
```

Comment:
I can't tell from this description what we expect to be happening in this spec. "properly" doesn't convey enough meaning without any other context.